src/test/java/com/exactpro/junit5/TestFormatter.java

```diff
@@ -0,0 +1,66 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestFormatter {
+
+    @Test
+    void testNameOneByte() throws IOException {
+        String hex = "12";
+        String methodName = new Object() {
```

Comment:
We pass it as string to a formatter. There is no need to use properly obtained name. The test will be easier to read if you just `String methodName = "aMethod"` and then `assertEquals("aMethod [ 12 ]", ..)`